modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc

```diff
@@ -4,12 +4,16 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates_{context}"]
-= Specifying host NICs in site PolicyGenTemplate CRs with hub cluster templates
+= Specifying group and site configuration in group PolicyGenTemplate CRs with hub templates
 
-You can manage host NICs in a single `ConfigMap` CR and use hub cluster templates to populate the custom NIC values in the generated polices that get applied to the cluster hosts.
-Using hub cluster templates in site `PolicyGenTemplate` (PGT) CRs means that you do not need to create multiple single site PGT CRs for each site.
+You can manage the configuration of fleets of clusters with `ConfigMap` CRs by using hub templates to populate the group and site values in the generated policies that get applied to the managed clusters.
+Using hub templates in site `PolicyGenTemplate` (PGT) CRs means that you do not need to create a `PolicyGenTemplate` CR for each site.
 
-The following example shows you how to use a single `ConfigMap` CR to manage cluster host NICs and apply them to the cluster as polices by using a single `PolicyGenTemplate` site CR.
+You can group the clusters in a fleet in various categories, depending on the use case, for example, hardware type or region. 
+Each cluster should have a label corresponding to the group(s) that the cluster is in. 
+If you manage the configuration values for each group in different `ConfigMap` CRs, then you require just one group `PolicyGenTemplate` CR to apply the changes to all the clusters in the group by using hub templates.
```

Comment:
Per the [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#j) entry on "just": "Omit just if the omission does not change the sentence meaning. Do not use just to mean only; use only instead."

If just here is used to mean only, I'd change it to say only:
```suggestion
If you manage the configuration values for each group in different `ConfigMap` CRs, then you require only one group `PolicyGenTemplate` CR to apply the changes to all the clusters in the group by using hub templates.
```